jenkins/run-server-func-tests

```diff
@@ -73,30 +88,25 @@ if [[ ${exit_status} -ne 0 ]]; then
     podman exec ${PB_POD_NAME}-pbenchserver journalctl
     printf -- "\n--- journalctl dump ---\n\n"
 fi
-if [[ -z ${1} ]]; then
-    echo "No clean up requested ... Pbench Server container and support services pod likely still running!"
+
+if [[ -z "${cleanup_flag}" ]]; then
+    echo "No clean up requested -- the Pbench Server container and support services pod are running!"
+    trap -
```

Comment:
Oops. You still need to specify the signals. `trap - INT QUIT TERM EXIT` will remove them all. I'm (almost, maybe, nearly) sorry I got you into this. Not really. Maybe. 👀

Reply:
Well, let's see what this next revision brings....  😆